agency-api-spec.md

```diff
@@ -200,20 +233,17 @@ provided below.
     "components": [
         {
             // ...
-            "additional_form_fields": [
+            "form_fields": [
```

Comment:
Did you see the sample JSON API output that I sent over in an e-mail? It's pretty different than what we have outlined here. With that being said, it was incomplete, as it contained some things we don't want, and was missing some items we do want (e.g. the form fields). We're getting close to working the ticket to figure out how to get the all of the right contents we want in the metadata file. I think once we have that, we can include it here as the sample metadata file.

Reply:
Yeah, since it was still pretty rough, I didn't want to include it yet and decided to base it on what we already have in the [schema](https://github.com/18F/foia-recommendations/pull/129/files) and the [initial form generation](https://raw.githubusercontent.com/18F/beta.foia.gov/master/www.foia.gov/_api/agencies/gsa.md). This is something I want to sync up on, so I'll grab some time from you.